### ISSUE TITLE
Node 0x3070f20f86fDa706Ac380F5060D256028a46eC29 raised {'code': -32000, 'message': 'call returned result of length 122944 exceeding limit 100000'}

### ISSUE THREAD

**Issue by cygnusv** [nucypher/nucypher#3394]
**Describe the Bug**
```
Node 0x3070f20f86fDa706Ac380F5060D256028a46eC29 raised {'code': -32000, 'message': 'call returned result of length 122944 exceeding limit 100000'}
```

**To Reproduce**
During TDec decryption, on mainnet ritual 1

**Traceback or Screenshots (Optional)**
If applicable, add the full error message / Traceback or upload screenshots to help explain your problem.

**System (please complete the following information):**
 - OS Version:
 - Nucypher Version:
 - Ethereum Node Version:

**Comment by cygnusv** [comment]
```
Traceback (most recent call last):
  File "/Users/david/dev/nu/nucypher/examples/testnet_simple_taco.py", line 93, in <module>
    cleartext = bob.threshold_decrypt(
  File "/Users/david/dev/nu/nucypher/nucypher/characters/lawful.py", line 734, in threshold_decrypt
    decryption_shares = self._get_decryption_shares(
  File "/Users/david/dev/nu/nucypher/nucypher/characters/lawful.py", line 661, in _get_decryption_shares
    raise Ursula.NotEnoughUrsulas(
nucypher.characters.lawful.Ursula.NotEnoughUrsulas: Threshold of Ursulas unable to decrypt: {'0x3070f20f86fDa706Ac380F5060D256028a46eC29': (<class 'nucypher.network.middleware.RestMiddleware.UnexpectedResponse'>, UnexpectedResponse("{'code': -32000, 'message': 'call returned result of length 122944 exceeding limit 100000'}"), <traceback object at 0x113585840>)}

```

**Comment by cygnusv** [comment]
A similar traceback from a different node:
```
}
2024-01-25T17:16:57+0000 [EventScannerTask#warn] Error during ritual event scanning: Traceback (most recent call last):
2024-01-25T17:16:57+0000 [EventScannerTask#warn] Restarting event scanner task!
2024-01-25T17:16:57+0000 [RitualTracker#debug] (0xFF8210) Scanning events in block range 52740790 - 52740816
2024-01-25T17:16:58+0000 [operator#debug] 0xFF8210 performing round 2 of DKG ritual #4 from blocktime 1706202991
2024-01-25T17:16:59+0000 [EventActuator#warn] Error during event hook: {'code': -32000, 'message': 'call returned result of length 121792 exceeding limit 100000'}
```
This one was using a getblock.io RPC provider.

**Comment by cygnusv** [comment]
Potentially caused by #3417

**Comment by derekpierre** [comment]
Closed via #3419
